<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionDialog</class>
 <widget class="QDialog" name="OptionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Serial Setting</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="10,2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="tabPosition">
      <enum>QTabWidget::North</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSerial">
      <attribute name="title">
       <string>Serial Port</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Port</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboPorts"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Baudrate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QComboBox" name="comboBaudrate">
         <property name="currentText">
          <string notr="true">9600</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">9600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">14400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">19200</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">28800</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">38400</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">57600</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">115200</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data bits</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="comboDatabits">
         <property name="currentText">
          <string notr="true">5</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">6</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">7</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">8</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Stop bits</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QComboBox" name="comboStopbits">
         <property name="currentText">
          <string notr="true">1</string>
         </property>
         <item>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">1.5</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string notr="true">2</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Parity</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QComboBox" name="comboParity">
         <property name="currentText">
          <string>None</string>
         </property>
         <item>
          <property name="text">
           <string>None</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Odd</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Even</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QPushButton" name="btnRefresh">
         <property name="text">
          <string>Refresh Ports</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabBle">
      <attribute name="title">
       <string>BLE</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1">
       <property name="leftMargin">
        <number>10</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QListWidget" name="listBLE"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnRefreshBLE">
           <property name="text">
            <string>Refresh Devices</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="lblBLEName">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Address</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="lblBLEAddr">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnConfirm">
       <property name="text">
        <string>Ok</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
